### DISCUSSION TITLE
Welcome to PLAYHYDRA Discussions!

### DISCUSSION THREAD

**Discussion by liamhtml** [liamhtml/PLAYHYDRA#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

**Comment by hgoodmanblair** [comment]
Hello. I don't really understand why you would IFRAME the games to your site, because now the games can get blocked, why not just upload the game files to your site so that they cannot be blocked?

**Reply by liamhtml** [reply]
Three reasons:
- generally it's significantly easier to iframe than to port the entire game, and when trying this I've been met with problems
- at least at my school the administration probably doesn't even know what HTML is and won't be able to figure out how to do this.
- if the iframe source url gets blocked it's very easy to just find another one

**Reply by ivysincredible** [reply]
sup, I had found ur playhydra project and have updated it a lot, here is my dc username: KermitTheFrog[#0096](https://www.youtube.com/hashtag/0096) I also friended u on dc, heres the link to the updated playhydra https://www.byronbonkers.me/

**Reply by ivysincredible** [reply]
> Three reasons:
> 
> * generally it's significantly easier to iframe than to port the entire game, and when trying this I've been met with problems
> * at least at my school the administration probably doesn't even know what HTML is and won't be able to figure out how to do this.
> * if the iframe source url gets blocked it's very easy to just find another one

pls can u friend me on dc, I have friended u and u msged me who are you but i can't message you because ur not my friend

**Comment by ivysincredible** [comment]
sup, I had found ur playhydra project and have updated it a lot, here is my dc username: KermitTheFrog[#0096](https://www.youtube.com/hashtag/0096) I also friended u on dc, heres the link to the updated playhydra https://www.byronbonkers.me
